src/VirtualClient/VirtualClient.Dependencies/MySqlServer/MySqlServerConfiguration.cs

```diff
@@ -113,6 +113,68 @@ public bool InMemory
         /// </summary>
         protected ISystemManagement SystemManager { get; }
 
+        /*
+                /// <summary>
+                /// Client used to communicate with the hosted instance of the
+                /// Virtual Client API at server side.
+                /// </summary>
+                public IApiClient ServerApiClient { get; set; }*/
+
+        /*
```

Comment:
Mark the PR as draft if it's still being tested

Reply:
These are to be removed. Updated. Thank you.